src/v2/view-builder/views/EnrollProfileView.js

```diff
@@ -5,15 +5,25 @@ import { FORMS as RemediationForms } from '../../ion/RemediationConstants';
 const Body = BaseForm.extend({
   title() {
     if (this.options.appState.getCurrentViewState().href.endsWith('idp/idx/enroll/update')) {
-      return loc('oie.primaryauth.submit', 'login');
+      const attributes = this.options.appState.attributes;
+      if (attributes?.uiDisplay?.label) {
+        return loc('oie.registration.form.customize.label', 'login', [attributes.uiDisplay.label]);
```

Comment:
do we really need to localize this? The values are dynamic, it looks odd to me to localize them.

Reply:
could you check the comments in the [ticket](https://oktainc.atlassian.net/browse/OKTA-462152), there is discussions about localization issue. Feel free to add your inputs! Thank you!

Reply:
I see now @fangshi-okta, one more question. This is only to avoid the english leak issue, right?

Reply:
Yeah that is why both of our string/boolean display mock json got English leak issue. Then I started to think about this problem!

Reply:
That is awesome @fangshi-okta